src/umpire/strategy/DynamicPool.cpp

```diff
@@ -115,7 +120,7 @@ DynamicPool::findFreeBlock(std::size_t bytes) const
   return iter;
 }
 
-void* DynamicPool::allocateFromResource(std::size_t bytes)
```

Comment:
`Resource` didn't make sense here, since `m_allocator` is an `AllocationStrategy`

Reply:
I guess it might be clearer if we could distinguish allocations from pool versus allocations from someplace else to grow the pool.  I don't like strategy since the pool is a strategy.  If resource doesn't work (I'm not sure why), then can you suggest a different name?  If not, then I would rather have it not change and remain resource.

Reply:
I switched these to `(de)allocateFromResource()`!

Reply:
How about `allocateBlock`?

Reply:
I like the suggestion from @davidbeckingsale!  Please change it to `(de)allocateBlock()`!